### PR TITLE
Added banner with version on scan

### LINKED ISSUE #16: Output a masthead and version for cargo-pants
Thanks for creating an issue! Please fill out this form so we can be
sure to have all the information we need, and to minimize back and forth.

* **What are you trying to do?**

We should bring this in to alignment with the other tools and output a:

```
cargo-pants
By Sonatype & Friends

cargo-pants version: CURRENT_VERSION
```

Screen Shot 2020-08-14 at 10 28 18 AM
^^^ screencap of Nancy

* **What feature or behavior is this required for?**

It's branding, someone in marketing will probably love this.

* **How could we solve this issue? (Not knowing is okay!)**

My gut says to look for a ascii art library and have some fun!

* **Anything else?**

This should be fun!

### PR DESCRIPTION
**Pull request by doddi**
This pull request makes the following changes:
* Outputs a banner during a scan
* Outputs version during a scan

Screenshot 2021-04-01 at 10 01 46

It relates to the following issue #s:
* Fixes #16

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -133,6 +135,11 @@ fn audit(lockfile_path: String, verbose_output: bool) -> ! {
     }
 }
 
+fn banner() {
+    println!("{}", std::include_str!("banner.txt"));
```
Comment (bhamail) [inline src/main.rs:139]: I'm on the fence about always showing the banner. Our other CLI apps tend to only show a banner when in "loud"/verbose mode, as users expressed a preference for "quiet by default". It's not a blocker for me, but maybe showing the banner only when the "loud"/v flag is present is a good compromise? Your call.

Either way, I really like the idea of always showing the name/version text, regardless of flags.

**Review by bhamail — APPROVED** [review]
LGTM. One question, but not a blocker as far as I'm concerned.

File: src/main.rs (hunk 1 above)
Comment (doddi) [reply src/main.rs:139]: ah ok cool, I didnt realise that was how it currently works on others. The raised issue didn't make that statement. I will update 👍

Comment (doddi) [reply src/main.rs:139]: https://github.com/sonatype-nexus-community/cargo-pants/pull/26/commits/a9e7c190443fffe5639e594bd696c37465931193

**Comment by bhamail** [comment]
Thanks @doddi !

**Pull request merged**